src/pykeen/experiments/rgcn/schlichtkrull2018_rgcn_fb15k.yaml

```diff
@@ -1,30 +1,68 @@
+# Official code: https://github.com/MichSchli/RelationPrediction
+# Arxiv: https://arxiv.org/abs/1703.06103, last update: Thu, 26 Oct 2017
+# Conference Proceddings: https://link.springer.com/chapter/10.1007/978-3-319-93417-4_38
 metadata:
   comments: ''
```

Comment:
are these empty comments necessary? same for the other files

Reply:
They came from converting the original JSON. Since they get converted to a dictionary upon loading, and the dictionary processing in the pipeline_from_config is a bit convoluted, I was not sure whether I would be allowed to remove them :slightly_smiling_face:

Reply:
You have the power. Be the positive force you want to see in this world.